<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>Computational Biology Seminar Series</title>
		<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css">
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,300,500,700' rel='stylesheet' type='text/css'>
		<link rel="stylesheet" href="css/style.css">
		<script>
		  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		  ga('create', 'UA-43635419-1', 'lsu.edu');
		  ga('send', 'pageview');

		</script>
	</head>
	<body class="container">
		<header>
			<img src="img/banner.jpg" class="img-rounded banner">
			<h2><strong>Computational Biology Seminar Series for Undergraduates</strong></h2>
			<p>
				<em>Sponsored by the <a href="http://science.lsu.edu/">LSU College of Science</a>, the <a href="http://www.biology.lsu.edu">Department of Biological
Sciences</a>, the <a href="http://cct.lsu.edu/">Center for Computation & Technology</a> and the <a href="http://lbrn.lsu.edu">Louisiana Biomedical Research Network</a></em>
			</p>
		</header>
				<div class="row">
			<div class="col-lg-8 col-md-8">
				<h3>A Survey of the SMR Gene Family in the Plant Kingdom, Using a Motif Oriented Search Tool</h3>
<!--				<p style="text-align: center">
					<iframe width="420" height="315" src="//www.youtube.com/embed/NY_K7lDXiM4" frameborder="0" allowfullscreen></iframe>
				</p>
-->

				<h4>Abstract:</h4>
				<p>Recent next-generation sequencing has made genomes available faster at an unprecedented rate. Consequently, efficiently searching for homologous genes in a wide array of species has become increasingly important. The BLAST algorithm has been widely used to identify homologous sequences in taxa, based on sequence similarity. However, BLAST recovers a high frequency of false positives when searching for highly diverse sequences. I have developed a protein sequence search method based on conserved motifs to detect homologous proteins with high sequence divergence. Initially, I developed this to search for the SMR protein family, cyclin-dependent kinase inhibitors unique to plants in cell cycle regulation and endoduplication. The SMR family of proteins cannot be identified through BLAST searches because the conserved sequences are limited to few short dispersed motifs. I searched for genome-wide SMR genes from over twenty diverse plant genomes using a Perl-based script that was developed to find conserved motifs in peptide sequences. The algorithm I developed accounts for empirical evidence of essential residues in defined motifs. A phylogenetic tree was created with RAxML based on a truncated Clustal Omega sequence alignment. Three deep clades with SMR genes were identified while some SMRs are only found in within the plant lineage Brassicaceae. I will be presenting the SMR family divergence with relevance to gene family duplications, motif order, and positive selection in 23 diverse plant lineages.</p>
<h4>Bio:</h4>
<p> My undergraduate career at LSU began with fixing computers as an IT intern in the Life Sciences Building. I spent a lot of time in diagnosing and fixing computer problems in various labs. Spending so much time in these labs peaked my interest in research and I began searching for a lab to join. The summer after my sophomore year, Dr. Dassanayake allowed me to gain experience working in her comparative genomics lab. Initially, I spent a lot of time planting and harvesting seeds but eventually I learned PCR and other wet lab techniques. With my background in computers and technology, I was assigned bioinformatic projects like building gene trees and writing scripts. With my research on the SIM/SMR (SIAMESE and SIAMESE-RELATED) gene-family, I have had the opportunity to give talks and present posters at various conferences including the PAG (Plant and Animal Genome) XXIV Conference in San Diego. 
</p>
				
			</div>
			<div class="col-lg-4 col-md-4">
				<img src="img/johnson.jpeg" class="pull-right">
				<h3><a href="http://www.phyleaux1.lsu.edu/?q=node/64">Lyndon M Coghill</a></h3>
				<p>Undergraduate Researcher, Biological Sciences, Louisiana State University</p>
				
	<!--		<script>
					var x     = new Date();
					var today = new Date();
					x.setFullYear(2016,3,14);
					if (today.getTime() == x.getTime()) {
					 	document.write('<div class="well"><h4>Live Feed:</h4><a href="https://seevogh.com/?join=6834267253" class="btn btn-primary"><img style="width: 20px" src="img/seevogh.png"> SeeVogh (<strong>HD</strong>)</a> </p> <p> No Access Code/No Login required, online link does not begin until 30 minutes before seminar does. If this is your first SeeVogh meeting follow this link for a brief tutorial: <a href="https://seevogh.com/how-it-works/tutorials/">https://seevogh.com/how-it-works/tutorials/</a> </p> <p> System Requirements: PCs running either Windows, Mac or Linux operating systems. </p> <p> <em>Note: if you plan to use an iOS or Android device, download the appropriate app via the http://seevogh.com/ website and then join the meeting # <strong>6834267253</strong> </em></p></div>');
					}
				</script> -->

			</div>
		</div>
	
		<ul class="pager">
			<li class="previous"><a href="http://lbrn.lsu.edu/events/comp-bio">&larr; Back to All Seminars</a></li>
		</ul>
		<script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
		<script src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
	</body>
</html>
